docs/src/assets/styles.css

```diff
@@ -546,7 +546,7 @@ div.note {
 }
 
 div.q-and-a {
-  background: #F5F5DC;
+  background: #F5DCDC;
```

Comment:
This is a change from [beige](https://www.colorhexa.com/f5f5dc) to [the below color](https://www.colorhexa.com/f5dcdc) for anyone curious.

![image](https://user-images.githubusercontent.com/43425812/151888890-52fb5a30-8805-42bb-9328-13c076875abb.png)